frontend/front/src/pages/Public/Layout/Navbar.js

```diff
@@ -72,7 +72,7 @@ function Navbar(props) {
 
   const open = Boolean(anchorMore);
 
-  const handleClickMore = (event) => setAnchorMore(event.currentTarget);
+  const handleClickMore = (event) => setAnchorMore(event?.currentTarget);
```

Comment:
Hey @mattdelsordo, you need to undo this part. The "Learn More" dropdown menu (desktop view) is unclickable.

Reply:
Ack, sorry! Should have tested this harder!

Reply:
Should be fixed now